infixInput.java

```diff
@@ -4,27 +4,115 @@
 
 public class infixInput {
 	public static void main(String[] args ) throws FileNotFoundException{
-		ArrayList<String> inputs = new ArrayList<String>();
-		inputs = readFile();
+		FileInputStream inputFile = new FileInputStream("input.txt");
+		Scanner scnr = new Scanner(inputFile);
 
-		for (int i = 0; i < inputs.size(); i++) {
-			System.out.println(inputs.get(i).toString());
+		String line = "";
+		while (scnr.hasNext()) {
+			line = scnr.nextLine();
+			System.out.println(line);
 		}
 	}
+	// iterate through the input file and get the first line(the line), call that
+	// line line
+	public static int getPredence(String A) { // sets the predence of the operators
+		switch
```

Comment:
take the switch out, sorry i forgot to take that out. i did this because i was thinking about changing it from if statements to a switch instead, because it would look cleaner